### PR TITLE
[cleanup] Modernize some code

### PR DESCRIPTION
**Pull request by hazendaz**

### CODE REVIEW

File: src/main/java/net/revelc/code/formatter/model/ConfigReader.java [hunk 1]
```diff
@@ -47,11 +47,10 @@ public Map<String, String> read(final InputStream input) throws IOException, SAX
         digester.addRuleSet(new RuleSet());
 
         final var result = digester.parse(input);
-        if (!(result instanceof Profiles)) {
+        if (!(result instanceof final Profiles profiles)) {
```
Comment (ctubbsii) [inline src/main/java/net/revelc/code/formatter/model/ConfigReader.java:50]: This is nice. Is this a newer JDK feature?

Comment (hazendaz) [reply src/main/java/net/revelc/code/formatter/model/ConfigReader.java:50]: https://docs.oracle.com/en/java/javase/17/language/pattern-matching-instanceof-operator.html

Comment (hazendaz) [reply src/main/java/net/revelc/code/formatter/model/ConfigReader.java:50]: I believe sonarlint calls that out and has auto fix for it.

Comment (ctubbsii) [reply src/main/java/net/revelc/code/formatter/model/ConfigReader.java:50]: The negation makes the scope where the pattern variable can be used a bit less intuitive, but overall, I like it.

**Pull request merged**